oie/ja-jp/Content/Topics/Apps/apps-about-oidc.htm

```diff
@@ -319,14 +319,14 @@ <h2>サービスプロバイダーとしての<span class="mc-variable okta-vari
             </li>
                                                         <li value="2">SPとしてのOktaが、ユーザー認証を外部IdPに委任します。
             </li>
-                                                        <li value="3">外部IdPが、Oktaの認可サーバーとして機能します。 </li>
+                                                        <li value="3">外部IdPが、OktaのAuthorization Serverとして機能します。 </li>
```

Comment:
Trust Device is translated to japanese. Authorization Server is translated to english. This Looks a bit random to me, but I also don't think we should be looking at those. We need to verify that pages are in correct state, html tags are not broken